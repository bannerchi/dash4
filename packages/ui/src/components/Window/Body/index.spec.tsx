/// <reference types="@types/jest" />
import { render } from '@testing-library/react';
import React from 'react';
import { WindowBody } from './index';

describe('WindowBody', () => {
	test('exist', () => {
		expect(typeof WindowBody).toBe('object');
	});
	test('render default', () => {
		const { container } = render(<WindowBody>Hello</WindowBody>);
		expect(container.firstChild).toMatchSnapshot();
	});
});
