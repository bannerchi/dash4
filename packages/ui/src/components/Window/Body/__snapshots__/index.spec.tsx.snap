// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WindowBody render default 1`] = `
.emotion-0 {
  padding: 5px;
  white-space: normal;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<div
  class=" emotion-0"
>
  Hello
</div>
`;
